VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/select-hearing-layout/select-hearing-layout.component.spec.ts

```diff
@@ -105,17 +105,18 @@ describe('SelectHearingLayoutComponent', () => {
         expect(videoCallService.updatePreferredLayout).toHaveBeenCalledWith(component.conference.id, layout);
     });
 
+    it('should return recommended layout', () => {
+        component.selectedLayout = undefined;
+        const layout = component.getSelectedOrPreferredLayout();
+        expect(layout).toBe(HearingLayout.OnePlus7);
```

Comment:
Why did this change from Dynamic to OnePlus7?

Reply:
Beause I added a staff member to the participants list and the recomand layout depends on total numbers of participants

Reply:
Is there test cases for different numbers of participants? This should really be separated from the ConferenceTestData if a change there will effect tests like this.

Reply:
I see there's test cases for the number of participants. This more tests that the correct methods are called. I'd be inclined to change this by spying on the relevant methods and ensure they are called with the correct values. Since this is a change to existing code I'll approve and leave it up to you if you want to update.

Reply:
I think there is no test cases for different numbers of participants